### PR TITLE
Second set of ad-accounts refactor changes

### PR DESCRIPTION
**Pull request by kreedula**
- Added additional constants and error strings as needed
- Added a new constructor to `BrokerTokenResponse` and unused remove primary refresh token param
- Removed redundant `IllegalArgumentException` from AcquireTokenParameters
- Added missing ResponseCodes to `MicrosoftStsAuthorizationFactory`

### CODE REVIEW

File: common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerTokenResponse.java [hunk 1]
```diff
@@ -37,16 +37,27 @@
...
 
     @SerializedName("not_before")
     private Date mExpiresNotBefore;
 
     @SerializedName("cloud_instance_host_name")
     private String mCloudInstanceHostName;
 
-    public BrokerTokenResponse(){
-
+    public BrokerTokenResponse(MicrosoftStsTokenResponse copy){
```
Comment (shoatman) [inline common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerTokenResponse.java]: I'd suggest an alternate parameter name.  tokenResponse?

File: common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerTokenResponse.java [hunk 2]
```diff
@@ -75,7 +85,7 @@ protected BrokerTokenResponse(Parcel in) {
     @Override
     public void writeToParcel(Parcel dest, int flags) {
         if (dest != null) {
-            dest.writeLong(getExpiresIn() != null ? getExpiresIn() : 0);
+            dest.writeLong(getExpiresIn());
```
Comment (shoatman) [inline common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerTokenResponse.java]: I'm assuming that getExpiresIn has been refactored to no longer return null?

Comment (kreedula) [reply common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerTokenResponse.java]: Thanks shane ,, this is an unintentional change, reverting it back

**Comment by kreedula** [comment]
Thanks @shoatman for the review. Addressed your comments

**Pull request merged**